tasks/managed/push-oot-kmods-to-git/push-oot-kmods-to-git.yaml

```diff
@@ -120,26 +126,153 @@ spec:
           --single-branch --branch "$(params.gitBranch)" \
           "$GIT_URL_WITH_AUTH" local-artifacts
         set -x # enable logging after the token is added to the URL
+
         cd local-artifacts
         git sparse-checkout init --cone
-        
-        # shellcheck source=/dev/null
-        . ../envfile
-        
-        TARGET_DIR="${DRIVER_VENDOR}_${DRIVER_VERSION}_${KERNEL_VERSION}"
-        git sparse-checkout set "${TARGET_DIR}"
-        git checkout
-        mkdir -p "${TARGET_DIR}"
-        cp ../*.ko "${TARGET_DIR}/"
-        cp ../signed_kmods_checksums.txt "${TARGET_DIR}/"
-        if [ -n "$(params.rpmPath)" ] && [ -d "${RPM_FULL_PATH}" ] && \
-           [ -n "$(find "${RPM_FULL_PATH}" -maxdepth 1 -name '*.rpm' -print -quit)" ]; then
-            cp "${RPM_FULL_PATH}"/*.rpm "${TARGET_DIR}/"
-        fi
         git config user.email "partner-accelerators@konflux-ci.redhat.com"
```

Comment:
wouldn't be better having this in a configmap or a secret? So you don't need to wait for a release cycle (2 weeks) in case for any reason the e-mail changes. - just a suggestion for improvement. Does not need to be in this PR.

Reply:
Probably it's better and we should change it but that code is already merged. Could we do that change in a different PR when we decide if we add a secret or a configmap?